### PR TITLE
Copy all Knowledge Center changes from Test env.

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
- Google Analytics
- New Footer w/ ACF Privacy Policy & Jump To Top link
- New Give Feedback form
- Updated content & new imagery on Complete Resubmission, Existing Login, New Login pages
- New (USWDS) icon-list based changelog on homepage

## How to Test
All changes are live in test environment: https://reitermb.github.io/Test/knowledge-center/

### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [x] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [x] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### CODE REVIEW

**Comment by ADPennington** [comment]
as soon as time allows, @reitermb @ttran-hub please pair on `gov a11y review`.

File: product-updates/knowledge-center/give-feedback.html [hunk 1]
```diff
@@ -265,7 +265,7 @@ <h1>Give Feedback</h1>
...
...scovered the issue? Please be specific. This information really helps us get to the bottom of an issue more quickly","isRequired":true},{"type":"text","name":"page","visibleIf":"{type} = 'Bug'","title":"What page of TDP were you on when it happened?"},{"type":"comment","name":"expectation","visibleIf":"{type} = 'Bug'","title":"What did you expect to happen instead?"},{"type":"comment","name":"setup-details","visibleIf":"{type} = 'Bug'","title":"Other helpful information:","description":"Your internet browser & operating system, whether you're able to reproduce the issue when you try again, etc..."},{"type":"comment","name":"improvement-idea","visibleIf":"{type} = 'Improvement'","title":"Tell us about your idea for improving TDP","isRequired":true},{"type":"boolean","name":"followup","visibleIf":"{type} anyof ['Feedback', 'Bug', 'Improvement']","title":"Would you like to provide your email address so we can follow up with you about your feedback?","defaultValue":"No","isRequired":true,"valueTrue":"Yes","valueFalse":"No"},{"type":"text","name":"email","visibleIf":"{followup} = 'Yes'","title":"What's your email address?","isRequired":true,"autoComplete":"email"}]}],"showTitle":false};
```
Comment (reitermb) [inline product-updates/knowledge-center/give-feedback.html:268]: ```"title":"Please rate your experience with the TANF Data Portal. (Higher is better)"```

Added to label cc: @ttran-hub

Comment (reitermb) [reply product-updates/knowledge-center/give-feedback.html:268]: Change is demoable in the test environment: https://reitermb.github.io/Test/knowledge-center/give-feedback.html

Comment (ttran-hub) [reply product-updates/knowledge-center/give-feedback.html:268]: > `"title":"Please rate your experience with the TANF Data Portal. (Higher is better)"`
> 
> Added to label cc: @ttran-hub

A11y 🚀

File: product-updates/knowledge-center/create-new-login.html [hunk 2]
```diff
@@ -285,7 +296,13 @@ <h2 class="usa-process-list__heading">Create your Account</h2>
           <li class="usa-process-list__item">
             <h2 class="usa-process-list__heading">Confirm your Email Address</h2>
             <p>
-              Check your email inbox for a message from Login.gov and select 'Confirm email address'. This will take you back to the Login.gov website.
+              Check your email inbox for a message from Login.gov and select 'Confirm email address'. This will take you back to the Login.gov website.<br>
+              <br>
+              If you did not receive an email, there are a few steps you can take:
+              <ul>
+                <li>Check your email’s spam or junk folder for an email from 'no-reply@tanfdata.acf.hhs.gov'.</li>
```
Comment (ADPennington) [inline product-updates/knowledge-center/create-new-login.html]: @reitermb the confirm email comes from login.gov. the emails from no-reply start after a user has requested access to TDP. can you move this reminder to the following steps: request access, approval, and data submitted?

also are we adding screenshots of denied, deactivation warnings, and deactivated emails anywhere in the knowledge center?

**Review by ADPennington — CHANGES_REQUESTED** [review]
@reitermb see [comment](https://github.com/raft-tech/TANF-app/pull/2220#discussion_r1011789456)

File: product-updates/knowledge-center/create-new-login.html (hunk 2 above)
Comment (ADPennington) [reply product-updates/knowledge-center/create-new-login.html]: @lilybl1 @reitermb @stevenino:

- @lilybl1 will fix where no-reply is located. 
- @lilybl1 will add a sentence about denial emails in italics under approval sections. but no pictures. 
- we need a new ticket to add pictures and description of denial, inactivity, and deactivation emails later after 2.2

I will approve this today with these changes.

Comment (ADPennington) [inline product-updates/knowledge-center/create-new-login.html]:
```suggestion
```

Comment (ADPennington) [reply product-updates/knowledge-center/create-new-login.html]: lets wait until @reitermb returns, hopefully tomorrow.

Comment (reitermb) [reply product-updates/knowledge-center/create-new-login.html]: @ADPennington @stevenino @lilybl1 

Playing a little catchup here but I /think/ all the discussed changes--both from Friday and yesterday--should now be committed.

File: product-updates/knowledge-center/uploading-data.html [hunk 3]
```diff
@@ -265,7 +276,11 @@ <h3 class="usa-process-list__heading">Submit your files</h3>
       <p>After uploading the files you want to submit, click the ‘Submit Data Files’ button. You should see a confirmation that your sections were submitted!</p>
       <a href="../img/data-files/4-success-message.png" data-lity data-lity-desc="Upload components showing success message above upload components reading 'Successfully submitted section(s): 1 on Wed Jun 15 2022'">
       <img src="../img/data-files/4-success-message.png" alt="Upload components showing success message above upload components reading 'Successfully submitted section(s): 1 on Wed Jun 15 2022'">
-        </a>
+      </a>
+      <p>Upon successful submission of your data files you will also receive an email confirmation from <em>no-reply@tanfdata.acf.hhs.gov<em> (example below).<p>
+      <a href="../img/getting-started/data-submitted.png" data-lity data-lity-desc="Email example of the successfully submitted data files by section">
+      <img src="../img/getting-started/data-submitted.png" alt="Email example of the successfully submitted data files by section">
```
Comment (ADPennington) [inline product-updates/knowledge-center/uploading-data.html:282]: lets eventually get that correct email template added here @reitermb @lilybl1 
after 2.2

knowctr

**Review by ADPennington — APPROVED** [review]
lgtm thanks! lets eventually update data submitted email template here cc: @ttran-hub 

@reitermb ready to merge!

**Pull request merged**